.github/workflows/build.yml

```diff
@@ -0,0 +1,290 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Build"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  tests:
+    name: "Test"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies: ["lowest", "highest"]
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        operating-system: ["ubuntu-latest", "windows-latest"]
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2.0.0"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@1.8.2"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+          extensions: mbstring
```

Comment:
If we require `mbstring`, this should probably be in our own `composer.json` :scream:

Reply:
@asgrim mbstring is only for phpunit. And this requirement will be removed with newer phpunit.

Reply:
Ahh gotcha, thanks for confirming!